modules/gltf/src/gltf/gltf-parser.js

```diff
@@ -120,6 +120,40 @@ export default class GLTFParser {
     return this._get('meshes', index);
   }
 
+  getDecompressedMesh(index) {
+    if (!this.DracoDecoder) {
+      throw new Error('DracoDecoder not available');
+    }
+
+    const mesh = this._get('meshes', index);
+
+    for (const primitive of mesh.primitives) {
+      // TODO: DracoMesh extension
+
+      const extensions = primitive.extensions;
+      if ('UBER_draco_point_cloud_compression' in extensions) {
```

Comment:
Since this function will grow to handle multiple extensions, put processing of each extension in separate "underscore" helper method?